src/openxr/OpenXRApi.cpp

```diff
@@ -313,7 +493,15 @@ const char *OpenXRApi::default_interaction_profiles_json = "[\n"
 														   "			},\n"
 														   "			{\n"
 														   "				\"set\": \"godot\",\n"
-														   "				\"action\": \"joystick_button\",\n"
+														   "				\"action\": \"menu_button\",\n"
+														   "				\"paths\": [\n"
+														   "					\"/user/hand/left/input/menu/click\",\n"
+														   // "					\"/user/hand/right/input/system/click\"\n" // binding this will crash SteamVR...
```

Comment:
yeah I noticed, but crashing instead of just giving an error when we're trying to register it (unless I'm not capturing it properly, the json is our own implementation, we're building a data structure)